sdgym/benchmark.py

```diff
@@ -44,6 +45,10 @@
     used_memory,
 )
 
+multiprocessing.set_start_method('spawn', force=True)
+multiprocessing.reduction.ForkingPickler.dumps = cloudpickle.dumps
```

Comment:
Is it risky to replace the `dumps` function? It seems like ForkingPickler's loads is just pickle's load so it's probably safe to replace that, but it has a custom dumps function
https://github.com/python/cpython/blob/ec89620e5e147ba028a46dd695ef073a72000b84/Lib/multiprocessing/reduction.py#L49-L54

Reply:
I don't think it's risky, their [docstring](https://github.com/cloudpipe/cloudpickle/blob/bc677daaf3018c96671d704a081ff0591eb0855e/cloudpickle/cloudpickle.py#L17) mentions that "It does not include an unpickler, as standard Python unpickling suffices.
". That means their pickling is pretty standard, so it's unlikely to cause issues.

I added a context manager just in case though, so the changes are localized. Let me know what you think.

Reply:
Out of curiosity, does it work if you only change the loads function but keep their dumps function? If not I think this is good

Reply:
No. It works the other way around, if you only change the `dumps` function, but I decided to change `loads` as well to be consistent.